src/app.js

```diff
@@ -40,18 +60,25 @@ function respondWithHtmlWrappedInGovukLayout(res, status, body) {
 }
 
 app
-  .get("/", (req, res) => {
+  .get("/", async (req, res) => {
     const prototypeFromReferrer = parsePrototypeFromReferrer(
       req.get("Referrer")
     );
 
+    const dynosQuantity = await getDynosQuantity(
+      prototypeFromReferrer
+    );
+
+    const bodyContent = bodyContentBasedOnDynos(
+      dynosQuantity
+    )
+
     respondWithHtmlWrappedInGovukLayout(
       res,
       200,
       `
       <h1 class="govuk-heading-l govuk-!-margin-top-9">This prototype is currently turned off</h1>
```

Comment:
I think we should vary the page title as well, and hide the restart button if it's already started - I don't think restarting will do anything if it's already running